src/dune_rules/modules.ml

```diff
@@ -920,6 +920,18 @@ let source_dirs =
     |> List.fold_left ~init:acc ~f:(fun acc f -> Path.Set.add acc (Path.parent_exn f)))
 ;;
 
+let entry_modules t =
+  List.filter
+    ~f:(fun m -> Module.visibility m = Public)
+    (match t.modules with
+     | Stdlib w -> Stdlib.lib_interface w |> Option.to_list
+     | Singleton m -> [ m ]
+     | Unwrapped m -> Unwrapped.entry_modules m
+     | Wrapped m ->
+       (* we assume this is never called for implementations *)
```

Comment:
This comment no longer makes sense right?

Reply:
I believe this comment refers to module implementations rather than virtual lib implementations. I'll leave it for now.